README.md

```diff
@@ -46,7 +46,17 @@ We will focus on building an open API first. Then developers can use the API to
 
 ## User stories so far
 
-Our goal is to keep things simple and not reinvent wheels. So far we have only two user roles: participants and chapter organizers.
+Our goal is to keep things simple and not reinvent wheels. So far we have only two user roles: participants and chapter organizers, both of which are users.
+
+### As a user
+
+- I can open a registration page where I can sign up with email password.
```

Comment:
email and password. Details about min requirements?

Reply:
I don't think they are necessary for now, we can start implementing something very minimal and then build on top of it. So I would not add no requirements now.